server_test.go

```diff
@@ -41,7 +41,7 @@ func TestInstallGem(t *testing.T) {
 	}
 	wantHistory := []string{
 		"sudo yum update -y",
-		"sudo yum install -y gcc-c++ patch readline readline-devel zlib zlib-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison sqlite-devel",
+		"sudo yum install -y epel-release gcc-c++ patch readline readline-devel zlib zlib-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison sqlite-devel",
```

Comment:
What do we need `epel-release` for?

Reply:
we don't need it since we're installing ruby via rvm.